mediapipe/examples/desktop/pose_tracking_dll/pose_tracking_lib.cpp

```diff
@@ -67,7 +67,7 @@ MPPoseTrackingDetector::InitPoseTrackingDetector(const char *pose_landmark_model
 }
 
 absl::Status
-MPPoseTrackingDetector::DetectPosesWithStatus(const cv::Mat &camera_frame, bool *isPose){
+MPPoseTrackingDetector::DetectPosesWithStatus(const cv::Mat &camera_frame, bool *isPose,  const std::chrono::milliseconds& timestamp){
```

Comment:
I wonder why do we pass milliseconds instead of microseconds from our library? timestamp from Jack's camera do not contain microseconds information?

Reply:
The camera contains the microseconds, but for some reason, we convert them to milliseconds and use milliseconds everywhere